onnxruntime/test/optimizer/initializer_test.cc

```diff
@@ -21,10 +21,10 @@ namespace test {
 namespace {
 template <typename T>
 Status WriteExternalDataFile(gsl::span<const T> data, const PathString& path, ScopedFileDeleter& file_deleter) {
-  std::vector<char> data_bytes(data.size_bytes());
-  ORT_RETURN_IF_ERROR(onnxruntime::utils::WriteLittleEndian(data, gsl::make_span(data_bytes)));
+  std::vector<uint8_t> data_bytes(data.size_bytes());
+  ORT_RETURN_IF_ERROR(onnxruntime::utils::WriteLittleEndian(data, gsl::make_span(data_bytes)));  // .data(), data_bytes.size())));
```

Comment:
is the trailing comment needed?

Reply:
No - was temporary testing code that I forgot to remove.